src/main/resources/xslt/outputs/fr/models.xsl

```diff
@@ -242,9 +240,8 @@
     </xd:doc>
     <xsl:template match="Instance//TableLoop" mode="model">
         <xsl:param name="source-context" as="item()" tunnel="yes"/>
-        <xsl:variable name="name">
-            <xsl:value-of select="enofr:get-name($source-context)"/>
-        </xsl:variable>
+        <xsl:variable name="name" select="enofr:get-name($source-context)"/>
+
         <xsl:element name="{enofr:get-name($source-context)}"/>
```

Comment:
TODO : The variable name should be re-use for building the name of xsl:element (line 245)